catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/plugins/ParentMetacardStreamCreationPlugin.java

```diff
@@ -108,7 +108,7 @@ private void submitParentCreateRequest(Context context, CreateRequest createRequ
         catalogFramework.create(createRequest)
                 .getCreatedMetacards()
                 .forEach(createdMetacard -> {
-                    LOGGER.info("created parent metacard: metacard={}", createdMetacard.getId());
+                    LOGGER.debug("created parent metacard: metacard={}", createdMetacard.getId());
```

Comment:
Looking up at the code above this, it looks like it will only ever get created with a single metacard, which is probably why they did it the way they did. However, I like your code better and it does the same exact thing, so I'll use that.